### ISSUE TITLE
Investigate new rule additions

### ISSUE THREAD

**Issue by ben-eb** [ben-eb/eslint-config-cssnano#2]
Looking into these two for sure;
- http://eslint.org/docs/rules/prefer-const
- http://eslint.org/docs/rules/no-extra-bind

Potentially should investigate any others on here:

http://eslint.org/docs/rules/

**Comment by clshortfuse** [comment]
Hi from ~3 years into the future! 😆

+1 on `prefer-const`. I'm trying to fix up some cssnano code and I was asking myself, "Why isn't eslint yelling at me right now?"

https://github.com/cssnano/cssnano/blob/0dfe3355951fa4a080a04dca34c6d99420def7ac/packages/postcss-merge-rules/src/index.js#L24

**Comment by ben-eb** [comment]
Yeah the codebase has a few inconsistencies like this one, unfortunately it's a lot of work to maintain a config which is why you haven't seen any activity here. These days I prefer to use https://github.com/xojs/xo and have some small tweaks to their ruleset rather than defining the whole thing from scratch.

I'm inactive for the time being for cssnano but it might be worth looking to migrate to xo or a similar shared config for a better developer experience. I would probably look at auto-formatting code on commit with prettier too.
